src/components/About/index.tsx

```
@@ -102,22 +94,19 @@ const listItem = (item: any) => {
                     className="rainbow-border m-2 mx-auto w-1/2 rounded-full bg-white p-4"
                 />
 
-                <h1 className="mx-auto mt-8 w-3/4 text-2xl font-black uppercase">{item.name}</h1>
+                <h1 className="mt-8 font-display text-2xl uppercase md:text-3xl">{item.name}</h1>
                 <p className="mb-4 text-sm font-black uppercase">{item.name}</p>
                 <div dangerouslySetInnerHTML={{ __html: item.description }} />
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)